app/src/main/java/com/asfoundation/wallet/ui/iab/LocalPaymentInteractor.kt

```diff
@@ -55,10 +55,10 @@ class LocalPaymentInteractor(private val deepLinkRepository: InAppDeepLinkReposi
     return (status == PENDING_USER_PAYMENT && type == "TOPUP") || (status == COMPLETED && (type == "INAPP" || type == "INAPP_UNMANAGED")) || status == FAILED || status == CANCELED || status == INVALID_TRANSACTION
   }
 
-  fun getCompletePurchaseBundle(isInApp: Boolean, merchantName: String, sku: String?,
+  fun getCompletePurchaseBundle(type: String, merchantName: String, sku: String?,
                                 scheduler: Scheduler,
```

Comment:
the readability in this method could be improved if we organize the arguments in a different way

Reply:
I’m only seeing changing the scheduler to last? is that it?